### ISSUE TITLE
Implement an Owner / Privileged user system for special commands

### ISSUE THREAD

**Issue by Vogel612** [Unihedro/JavaBot#37]
It would be nice if "disruptive" commands like summon, unsummon and shutdown could only be executed by special people. These users should be maintained at runtime and persisted to the configuration, so as to be the same after restarts and the like.

**Comment by Vincentyification** [comment]
Good idea. Timed ownership will be effective as well, especially during testing.
